BLocal.Web.Manager.MVC3/Web.config.template

```diff
@@ -26,9 +26,9 @@
     <add name="iChoosr - Test" color="#f90">
       <valueProvider type="BLocal.Web.Manager.Providers.RemoteAccess.RemoteAccessManager, BLocal.Web.Manager, Version=1.0.0.0, Culture=neutral">
         <!-- Add constructor parameters -->
-        <add name="targetUrl" value="http://10.120.3.199:7000/ExternalSynchronization/" />
-        <add name="targetPassword" value="left-out-for-safety"/>
-        <add name="targetProviderPair" value="iChoosr - Test"/>
+        <add name="targetUrl" value="http://10.120.3.199:7000/RemoteAccess/" />
```

Comment:
@KrzychuR ExternalSynchronization -> RemoteAccess & targetProviderPair -> targetProviderGroup. Shouldn't there also be un update to code for these changes to work?

Reply:
@rusln I didn't found /ExternalSynchronization endpoint on our prod/test env. I believe it is mistake/old config on Git :(